src/registrar/admin.py

```diff
@@ -428,6 +431,26 @@ def get_ordering(self, request, queryset):
 
         return ordering
 
+    def get_filters_params(self, params=None):
+        """
+        Overrides the default behavior which gets filter_params, except
```

Comment:
Nitpick: Simplify that comment a bit: add portfolio to ignored params to allow the portfolio filter while not listing it as a filter option on the right side of Change List on the portfolio list.